packages/info-sprinkle/src/InfoSprinkle/InfoSprinkle.tsx

```diff
@@ -29,17 +29,20 @@ export const InfoSprinkle = React.forwardRef<
         darkMode={darkMode}
         baseFontSize={baseFontSize}
         trigger={
-          <span
-            tabIndex={0}
-            role="button"
+          <button
```

Comment:
https://github.com/mongodb/leafygreen-ui/pull/1999#discussion_r1347643174. We used `span` because of an example we found, but that example no longer uses a `span`; it uses a `button`. I couldn't find a reason to use a `span` so I swapped it with a `button`